<launch>
  <arg name="path_to_gt_map" default=""/>
  <arg name="path_to_save_graph" default="/home/kirill/TopoSLAM/toposlam_ws/src/simple_toposlam_model/graph_jsons_new/scout_rosbag"/>
  <arg name="path_to_save_logs" default="/home/kirill/TopoSLAM/toposlam_ws/data/prism_topomap_logs"/>
  <arg name="config_file" default="scout_rosbag.yaml"/>

  <node pkg="prism_topomap" type="prism_topomap_node.py" name="prism_topomap_node" output="screen">
    <param name="path_to_gt_map" value="$(arg path_to_gt_map)"/>
    <param name="path_to_save_graph" value="$(arg path_to_save_graph)"/>
    <param name="config_file" value="$(arg config_file)"/>
  </node>

  <node pkg="prism_topomap" type="tf_to_odom.py" name="odometry_publisher" output="screen"/>

  <!-- node pkg="tf" type="static_transform_publisher" name="tf_base_to_lidar" args="-0.300 0.014 0.883 -0.008 0.004 -0.008 1.000 base_link velodyne 100"/ -->

  <!-- node pkg="image_transport" type="republish" name="decompress_front_image" args="compressed in:=/zed_node/left/image_rect_color raw out:=/zed_node/left/image_rect_color"/ -->
  <node pkg="image_transport" type="republish" name="decompress_realsense_image" args="compressed in:=/camera/color/image_raw raw out:=/camera/color/image_raw"/>
</launch>